app/models/video.rb

```diff
@@ -1,11 +1,19 @@
 class Video < ActiveRecord::Base
 
+  has_many :reviews, ->{ order("created_at DESC") }
+  has_many :queue_items, ->{ order(:list_position)}
   belongs_to :category
-  scope :sorted, ->{order(:title)}
+  scope :sorted, ->{ order(:title) }
   validates_presence_of :title, :description
 
   def self.search_by_title(title)
     return [] if title.blank?
     self.where("title ILIKE ?", "%#{title}%").order("created_at DESC")
   end
+
+  def average_rating
```

Comment:
Nice! How'd you find this? Used the average method: 

def average_rating
    return 3.0 if self.reviews.count == 0
    self.reviews.average(:rating).round(2)
  end